src/quo/components/wallet/summary_info/style.cljs

```diff
@@ -4,7 +4,7 @@
 
 (defn container
   [networks? theme]
-  {:width         335
+  {:width         "100%"
```

Comment:
How about `:flex 1` ?

Reply:
For some reason it breaks the component, but full width works fine

![](https://github.com/status-im/status-mobile/assets/18485527/e0b73e26-28c2-4cb2-866e-6a73c8b46869)